src/controller/java/AndroidDeviceControllerWrapper.h

```diff
@@ -217,6 +220,8 @@ class AndroidDeviceControllerWrapper : public chip::Controller::DevicePairingDel
     // TODO: This may need to be injected as a SessionKeystore*
     chip::Crypto::RawKeySessionKeystore mSessionKeystore;
 
+    chip::app::DefaultICDClientStorage mICDClientStorage;
```

Comment:
if we reboot android chip-tool, can it be persistent?

Reply:
Yes. 
In each case, I confirmed that it was saved and loaded.
  - Java controller: File
  - Android: Shared Preference.

Reply:
thanks